### DISCUSSION TITLE
need implementing KODI Video Versions feature

### DISCUSSION THREAD

**Discussion by RealSting** [Komet/MediaElch#1744] [Ideas]
Need implementing Versions feature of KODI
I like to update KODI by remote (synchronization feature)
https://kodi.wiki/view/Video_versions
please :)

**Comment by bugwelle** [comment]
Cool! I didn't know Kodi supports that now. I'll transform this discussion into a feature request. :)

**Comment by bugwelle** [comment]
I opened https://github.com/Komet/MediaElch/issues/1745 ; discussion can continue there. :)
Thanks for opening this discussion!
